user_tools/src/spark_rapids_pytools/resources/dataproc-configs.json

```diff
@@ -258,6 +258,14 @@
       ]
     }
   },
+  "clusterInference": {
+    "defaultCpuInstances": {
+      "driver": "n1-standard-16",
+      "executor": {
+        "n1-standard": {"vCPUs": [1, 2, 4, 8, 16, 32, 64, 96]}
+      }
+    }
+  },
```

Comment:
We may need to support different families. Just asking here if the code can handle different type of instanceTypes for executors. Especially, when we support different GPUs like L4 which may be attached to different machine types.

Reply:
Yes, we can add another key such as `"n1-highmem"` to support multiple machine types.

Reply:
ok